src/app/[locale]/(houseHold)/[houseHoldId]/(content)/settings/page.tsx

```diff
@@ -9,10 +20,69 @@ export default async function HouseholdEditPage({
 }) {
   const { householdId } = await params;
   const household = await getHousehold(householdId);
+  const categories = await getCategories(householdId);
+  const currencies = await getCurrencies();
+  let members = await getMembers(householdId);
+  const session = await auth();
 
-  if (household == null) notFound();
+  members = members.filter(
+    (member) =>
+      member.user?.id !== session?.user.id &&
+      member.user?.id !== household?.ownerId
+  );
```

Comment:
jest jakiś konkretny powód dlaczego użytkownika tu filtrujemy?

może to być mylące jeśli gdzieś niżej sprawdzałbyś czy ten user na pewno jest członkiem gospodarstwa

jeśli to tylko po to żeby nie wyświetlać ikonki usera niżej to lepiej tam tą logikę przenieść

no i pytanko czemu ownera też tam skipujesz?